### PR TITLE
[#49] Added microbenchmark tutorial.

### PR DESCRIPTION
**Pull request by 365andreas**

### CODE REVIEW

File: src/Cardano/BM/Observer/Monadic.lhs [hunk 1]
```diff
@@ -35,6 +35,85 @@ import           Cardano.BM.Trace (Trace, logError, logNotice, subTrace, traceNa
...
+\newline
+\par\noindent
+1. Create a |trace| which will have been configured
+   to observe things besides logging.
+
+\begin{spec}
+        import qualified Cardano.BM.Configuration.Model as CM
+        . . .
+        c <- config
+        trace@(ctx, _) <- setupTrace (Right c) "demo-playground"
```
Comment (CodiePP) [inline src/Cardano/BM/Observer/Monadic.lhs]: `trace` alonw would be sufficient?

File: src/Cardano/BM/Observer/Monadic.lhs [hunk 2]
```diff
@@ -35,6 +35,85 @@ import           Cardano.BM.Trace (Trace, logError, logNotice, subTrace, traceNa
...
+\end{spec}
+
+2. |c| is the |Configuration| of |trace|. In order to
+   enable the collection and processing of measurements
+   (min, max mean, std-dev) |AggregationBK| is needed.
+
+\begin{spec}
+        CM.setDefaultBackends c [KatipBK, AggregationBK]
+\end{spec}
+means
```
Comment (CodiePP) [inline src/Cardano/BM/Observer/Monadic.lhs]: .. in a configuration file (YAML)

File: src/Cardano/BM/Observer/Monadic.lhs [hunk 3]
```diff
@@ -35,6 +35,85 @@ import           Cardano.BM.Trace (Trace, logError, logNotice, subTrace, traceNa
...
+
+4. Find an action to measure. e.g.:
+
+\begin{spec}
+        runProtocolWithPipe x hdl proto `catch` (\ProtocolStopped -> return ())
+\end{spec}
+
+    and use |bracketObserveIO|
+
+    e.g.:
```
Comment (CodiePP) [inline src/Cardano/BM/Observer/Monadic.lhs]: add `\\` to force a new line

File: src/Cardano/BM/Observer/Monadic.lhs (hunk 1 above)
Comment (365andreas) [reply src/Cardano/BM/Observer/Monadic.lhs]: No `ctx` is used 
https://github.com/input-output-hk/iohk-monitoring-framework/blob/3ad5ef61d5cddd5846c5530945f376559b7e813a/src/Cardano/BM/Observer/Monadic.lhs#L91-L94

Comment (CodiePP) [reply src/Cardano/BM/Observer/Monadic.lhs]: and there is `c` ;-)

**Pull request merged**